ast/src/parsed/display.rs

```diff
@@ -195,40 +181,8 @@ impl Display for InstructionParam {
     }
 }
 
-impl<T: Display> Display for ASMStatementBatch<T> {
-    fn fmt(&self, f: &mut Formatter<'_>) -> Result {
-        for s in &self.statements {
-            writeln!(f, "{s}")?;
-        }
-        write!(
-            f,
-            "// END BATCH{}",
-            self.reason
-                .as_ref()
-                .map(|reason| format!(" {reason}"))
-                .unwrap_or_default()
-        )
-    }
-}
-
-impl Display for Incompatible {
-    fn fmt(&self, f: &mut Formatter<'_>) -> Result {
-        write!(f, "{:?}", self)
-    }
-}
-
-impl Display for IncompatibleSet {
-    fn fmt(&self, f: &mut Formatter<'_>) -> Result {
-        write!(
-            f,
-            "{}",
-            self.0
-                .iter()
-                .map(|r| r.to_string())
-                .collect::<Vec<_>>()
-                .join(", ")
-        )
-    }
+pub fn quote(input: &str) -> String {
+    format!("\"{}\"", input.replace('\\', "\\\\").replace('"', "\\\""))
```

Comment:
Wasn't this added in a different PR?

Reply:
I moved it here from `parser_utils` as it's only used here